app/src/main/java/com/epfl/beatlink/ui/search/components/Cards.kt

```diff
@@ -0,0 +1,189 @@
+                .padding(12.dp)) {
+          // First Row: Icon + Title and Username
+          Row(
+              verticalAlignment = Alignment.CenterVertically,
+              modifier = Modifier.testTag("partyCardRow").fillMaxWidth()) {
+                // Icon on the left
+                CombinedPartyIcon()
+
+                // Title and Username
+                Column(
+                    verticalArrangement = Arrangement.SpaceBetween,
+                    modifier =
+                        Modifier.testTag("partyCardTextColumn")
+                            .height(28.dp) // Match icon's height
+                            .padding(horizontal = 6.dp)) {
+                      Text(
+                          text = title,
+                          fontSize = 14.sp,
+                          fontWeight = FontWeight.Bold,
+                          maxLines = 1,
+                          modifier = Modifier.testTag("partyCardTitle"))
+                      Text(
+                          text = username,
+                          fontSize = 10.sp,
+                          color = Color.Gray,
+                          maxLines = 1,
+                          modifier = Modifier.testTag("partyCardUsername"))
+                    }
+              }
+
+          // Second Row: Description
+          Text(
+              text = description,
+              fontSize = 10.sp,
+              color = Color.DarkGray,
+              maxLines = 2,
+              modifier =
+                  Modifier.testTag("partyCardDescription").fillMaxWidth().padding(top = 5.dp))
+        }
+  }
+}
+
+@Composable
+fun SongCard(song: SpotifyTrack) {
+  Card(
+      modifier = Modifier.testTag(song.name + "songCardItem").width(65.dp).height(85.dp),
+      shape = RoundedCornerShape(1.dp)) {
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            modifier =
+                Modifier.testTag(song.name + "songCardColumn")
+                    .fillMaxSize()
+                    .background(color = Color.White)) {
+              Card(
+                  modifier = Modifier.testTag(song.name + "songCardContainer").size(65.dp),
+                  shape = RoundedCornerShape(5.dp)) {
+                    Image(
+                        painter = painterResource(id = R.drawable.cover_test1),
+                        contentDescription = null, // Provide a description for accessibility
+                        modifier =
+                            Modifier.testTag(song.name + "songCardImage")
+                                .fillMaxSize() // Make image fill the card
+                        )
+                  }
+
+              Box(
+                  modifier =
+                      Modifier.testTag(song.name + "songCardTextBox")
+                          .width(53.dp)
+                          .height(20.dp)
+                          .background(color = Color.White)) {
```

Comment:
here you should use the app backgroup (eg. lightThemeBackground) instead of plain white